### DISCUSSION TITLE
Settting both customManagers and regexManagers silently ignores the latter

### DISCUSSION THREAD

**Discussion by astellingwerf** [renovatebot/renovate#32075] [Request Help]
### How are you running Renovate?

A Mend.io-hosted app

### Please tell us more about your question or problem

If a configuration has both `customManagers` and `regexManagers` attributes, Renovate will silently ignore the latter. One way to demonstrate this is to enable configuration migration: https://github.com/astellingwerf/renovate-migrate-customManagers-and-regexManagers/pull/1 shows that the migrated object simply has no `regexManagers`, but they're not translated into `customManagers` with `"customType": "regex"` either.

### Logs (if relevant)

<details><summary>Logs</summary>

```

Replace this text with your logs, between the starting and ending triple backticks

```

</details>

**Comment by rarkins** [comment]
Duplicate of #32020
